lib/fungus_toast/games/helpers/player_skills.ex

```diff
@@ -0,0 +1,162 @@
+defmodule FungusToast.PlayerSkills do
+  @moduledoc """
+  The PlayerSkills helper for the Games context.
+  """
+
+  import Ecto.Query, warn: false
+  alias FungusToast.Repo
+
+  alias FungusToast.Games
+  alias FungusToast.Games.{Player, PlayerSkill, Skill}
+
+  @doc """
+  Returns the skill information for a given player.
+
+  ## Examples
+
+      iex> get_player_skill(player, 1)
+      %PlayerSkill{}
+
+      iex> get_player_skill(1, 1)
+      %Player{}
+
+  """
+  def get_player_skill(%Player{} = player, skill_id) do
+    get_player_skill(player.id, skill_id)
+  end
+  def get_player_skill(player_id, skill_id) do
+    from(ps in PlayerSkill, where: ps.player_id == ^player_id and ps.skill_id == ^skill_id) |> Repo.one()
+  end
+
+  @doc """
+  Returns information about all skills for a given player.
+
+  ## Examples
+
+      iex> get_player_skills(player)
+      [%PlayerSkill{}, ...]
+
+      iex> get_player_skills(1)
+      [%Player{}, ...]
```

Comment:
Same question as above (regarding types)

Reply:
Should also be `%PlayerSkill{}`